app/assets/javascripts/reactions.js

```diff
@@ -0,0 +1,40 @@
+function changeReactionCount(value, warpableId) {
+  $('#reaction-count-' + warpableId).html(value);
```

Comment:
Can we use classes here, in case we start wanting to have the star buttons shown in multiple places? Like, for example, on the toolbars that popup on the images, for example. Thanks!

Reply:
I'm unable to add the functions in a class here..The class was not getting loaded. 
It said 'Reaction is undefined'
But I don't see why we need a class for this?
The functions are publicly accessible so they can be accessed from anywhere